go-controller/pkg/node/ovn_test.go

```diff
@@ -10,15 +10,18 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/client-go/kubernetes/fake"
 	"k8s.io/client-go/tools/record"
+
+	egressfirewallfake "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/crd/egressfirewall/v1/apis/clientset/versioned/fake"
 )
 
 type FakeOVNNode struct {
-	node       *OvnNode
-	watcher    *factory.WatchFactory
-	stopChan   chan struct{}
-	recorder   *record.FakeRecorder
-	fakeClient *fake.Clientset
-	fakeExec   *ovntest.FakeExec
+	node             *OvnNode
+	watcher          *factory.WatchFactory
+	stopChan         chan struct{}
+	recorder         *record.FakeRecorder
+	fakeClient       *fake.Clientset
+	fakeEgressClient *egressfirewallfake.Clientset
```

Comment:
Why is this actually even needed on the node side? You don't have any code touching egress firewall on the node side.